erdjava/src/test/java/elrond/TransactionTest.java

```diff
@@ -0,0 +1,70 @@
+package elrond;
+
+import org.junit.Test;
+import java.math.BigInteger;
+
+import static org.junit.Assert.assertEquals;
+
+public class TransactionTest {
+    @Test
+    public void shouldSerialize() throws Exception {
+        Transaction transaction = new Transaction();
+
+        // Without data (memo) field
+        transaction.setNonce(0);
+        transaction.setValue(new BigInteger("42"));
+        transaction.setSender(Address.fromBech32("erd1l453hd0gt5gzdp7czpuall8ggt2dcv5zwmfdf3sd3lguxseux2fsmsgldz"));
+        transaction.setReceiver(Address.fromBech32("erd1cux02zersde0l7hhklzhywcxk4u9n4py5tdxyx7vrvhnza2r4gmq4vw35r"));
+        transaction.setGasPrice(1000000000);
+        transaction.setGasLimit(50000);
+        transaction.setChainID("1");
+
+        String expected = "{\"nonce\":0,\"value\":\"42\",\"receiver\":\"erd1cux02zersde0l7hhklzhywcxk4u9n4py5tdxyx7vrvhnza2r4gmq4vw35r\",\"sender\":\"erd1l453hd0gt5gzdp7czpuall8ggt2dcv5zwmfdf3sd3lguxseux2fsmsgldz\",\"gasPrice\":1000000000,\"gasLimit\":50000,\"chainID\":\"1\",\"version\":1}";
```

Comment:
you can use:
`String myString = "using ?question mark? instead of quotes".replace('?', '"');` in order to refactor the double quotes encoding. You can use the backtick character but github would have formatted the comment if I would have used it